bouncer/shared/upgrade_network.ts

```diff
@@ -68,22 +73,34 @@ async function incompatibleUpgradeNoBuild(
 
   console.log('Engines started');
 
-  await submitRuntimeUpgradeWithRestrictions(runtimePath, undefined, undefined, true);
+  await submitRuntimeUpgradeWithRestrictions(runtimePath, undefined, undefined, false);
```

Comment:
Is this on purpose? I know you told me to change that to run it locally, but is it correct to change it here? Just checking it hasn't accidentally slipped into the PR.